projects/hip/docs/what_is_hip.rst

```diff
@@ -8,39 +8,37 @@
 
-The Heterogeneous-computing Interface for Portability (HIP) API is a C++ runtime API
-and kernel language that lets developers create portable applications running in heterogeneous systems,
-using CPUs and AMD GPUs or NVIDIA GPUs from a single source code. HIP provides a simple
-marshalling language to access either the AMD ROCM back-end, or NVIDIA CUDA back-end,
-to build and run application kernels.
+The Heterogeneous-computing Interface for Portability (HIP) API is a C++ runtime
+API and kernel language that lets developers create portable applications
+running in heterogeneous systems, using CPUs and AMD GPUs from a single source
+code.
 
 .. figure:: data/what_is_hip/hip.svg
     :alt: HIP in an application.
     :align: center
 
 * HIP is a thin API with little or no performance impact over coding directly
-  in NVIDIA CUDA or AMD :doc:`ROCm <rocm:what-is-rocm>`.
+  in AMD :doc:`ROCm <rocm:what-is-rocm>`.
 
 * HIP enables coding in a single-source C++ programming language including
   features such as templates, C++11 lambdas, classes, namespaces, and more.
 
-* Developers can specialize for the platform (CUDA or ROCm) to tune for
-  performance or handle tricky cases.
+* Developers can specialize for the platform to tune for performance or handle
+  tricky cases.
 
-ROCm offers compilers (``clang``, ``hipcc``), code
-profilers (``rocprof``, ``omnitrace``), debugging tools (``rocgdb``), libraries
-and HIP with the runtime API and kernel language, to create heterogeneous applications
-running on both CPUs and GPUs. ROCm provides marshalling libraries like
+ROCm offers compilers (``clang``, ``hipcc``), code profilers (``rocprof``,
+``omnitrace``), debugging tools (``rocgdb``), libraries and HIP with the runtime
+API and kernel language, to create heterogeneous applications running on both
+CPUs and GPUs. ROCm provides marshalling libraries like
 :doc:`hipFFT <hipfft:index>` or :doc:`hipBLAS <hipblas:index>` that act as a
-thin programming layer over either NVIDIA CUDA or AMD ROCm to enable support for
-either back-end. These libraries offer pointer-based memory interfaces and are
-easily integrated into your applications.
+thin programming layer over AMD ROCm and offer API compatiblity with CUDA
+equivalent library. These libraries offer pointer-based memory interfaces and
+are easily integrated into your applications.
 
 HIP supports the ability to build and run on either AMD GPUs or NVIDIA GPUs.
 GPU Programmers familiar with NVIDIA CUDA or OpenCL will find the HIP API
-familiar and easy to use. Developers no longer need to choose between AMD or
-NVIDIA GPUs. You can quickly port your application to run on the available
-hardware while maintaining a single codebase. The :doc:`HIPify <hipify:index>`
+familiar and easy to use. You can quickly port your application to run on the
+available hardware while maintaining a single codebase. The :doc:`HIPify <hipify:index>`
```

Comment:
This idea that HIP runs on NVIDIA GPUs was the thing I thought Saad was trying to eliminate. Check with him, but another approach is that HIP lets you quickly port CUDA code to run on AMD GPUs. This supports the "available hardware" theory, because your CUDA code runs on NVIDIA GPUs and your HIP code runs on AMD GPUs.